src/test_l1_tvSettings.c

```diff
@@ -12701,9 +12701,10 @@ void test_l1_tvSettings_negative_SaveDvTmaxValue (void)
  * | Variation / Step | Description | Test Data | Expected Result | Notes |
  * | :-------: | ------------- | --------- | --------------- | ----- |
  * | 01 | call TvInit() -  Initialise and get a valid instance of the TV client | void | tvERROR_NONE | Should Pass |
- * | 02 | call GetSupportedComponentColor() -  Retrieve the current Supported ComponentColor and validate Supported ComponentColor by looping through the ComponentColor section of test specific config file | int * | (tvERROR_NONE | tvERROR_OPERATION_NOT_SUPPORTED) | Should Pass |
- * | 03 | call GetSupportedComponentColor() -  Retrieve the current Supported ComponentColor with valid argument and validate with above value | int * | tvERROR_NONE | Should Pass |
- * | 04 | call TvTerm() -  Terminate and close the instance of the TV client | void | tvERROR_NONE | Should Pass |
+ * | 02 | call SetCMSState() - Set the CMSState with valid value | true | tvERROR_NONE | Should Pass |
+ * | 03 | call GetSupportedComponentColor() -  Retrieve the current Supported ComponentColor and validate Supported ComponentColor by looping through the ComponentColor section of test specific config file | int * | (tvERROR_NONE | tvERROR_OPERATION_NOT_SUPPORTED) | Should Pass |
```

Comment:
test_l1_tvSettings_positive_GetCurrentVideoResolution  - GetCurrentVideoResolution -The API returns the default value as tvVideoResolution_NONE, which needs to be validated. In interface document it is stated as the default output parameter.

Same way

test_l1_tvSettings_positive_GetCurrentVideoFrameRate  - GetCurrentVideoFrameRate - default out param is tvVideoFrameRate_NONE

test_l1_tvSettings_positive_GetCurrentVideoSource - GetCurrentVideoSource - default out param is VIDEO_SOURCE_IP

test_l1_tvSettings_positive_GetCurrentVideoFormat  - GetCurrentVideoFormat - default out param is VIDEO_FORMAT_SDR

Reply:
Updated.